examples/lsobj/lsobj.go

```diff
@@ -5,10 +5,11 @@ package main
 import (
 	"errors"
 	"fmt"
+	"log"
 	"os"
 	"unsafe"
 
-	"github.com/hillu/go-ntdll"
+	"github.com/oec/go-ntdll"
```

Comment:
Need to flip this back to hillu, eventually

Reply:
yeah, let me flip that back. needed it to test.

Reply:
reverted and checked in a new patch